AmplifyTests/CategoryTests/DataStore/Model/LazyReferenceTests.swift

```diff
@@ -84,4 +84,23 @@ final class LazyReferenceTests: XCTestCase {
         let json = try comment.toJSON()
         XCTAssertEqual(json, "{\"post\":[{\"name\":\"id\",\"value\":\"postId\"}],\"id\":\"commentId\",\"content\":\"content\",\"updatedAt\":null,\"createdAt\":null}")
     }
+    
+    func testDecodePrimaryKeysOnly() async throws {
```

Comment:
Conventional test name should include the expectation:
`testDecodePrimaryKeysOnlyNotLoaded`